skia-safe/src/modules/svg.rs

```diff
@@ -221,6 +221,100 @@ impl Dom {
         }
         Data::new_empty()
     }
+
+    pub fn root(&self) -> DomSVG {
+        DomSVG::from_unshared_ptr(unsafe { sb::C_SkSVGDOM_getRoot(self.native()) }).unwrap()
+    }
+}
+
+pub type DomSVG = RCHandle<sb::SkSVGSVG>;
+require_base_type!(sb::SkSVGSVG, sb::SkSVGContainer);
+unsafe_send_sync!(DomSVG);
+
+impl NativeRefCounted for sb::SkSVGSVG {
+    fn _ref(&self) {
+        unsafe { sb::C_SkSVGSVG_ref(self) }
+    }
+
+    fn _unref(&self) {
+        unsafe { sb::C_SkSVGSVG_unref(self) }
+    }
+
+    fn unique(&self) -> bool {
+        unsafe { sb::C_SkSVGSVG_unique(self) }
+    }
+}
+
+impl DomSVG {
+    pub fn intrinsic_size(&self) -> Size {
+        Size::from_native_c(unsafe { sb::C_SkSVGSVG_intrinsicSize(self.native()) })
+    }
+
+    pub fn set_attribute(&mut self, attribute: impl AsRef<str>, value: impl AsRef<str>) -> bool {
+        let attribute = CString::new(attribute.as_ref()).unwrap();
+        let value = CString::new(value.as_ref()).unwrap();
+        unsafe {
+            sb::C_SkSVGSVG_parseAndSetAttribute(
+                self.native_mut(),
+                attribute.as_ptr(),
+                value.as_ptr(),
+            )
+        }
+    }
+
+    pub fn set_x(&mut self, x: DomSVGLength) {
+        unsafe {
+            sb::C_SkSVGSVG_setX(self.native_mut(), *x.native());
+        }
+    }
+
+    pub fn set_y(&mut self, y: DomSVGLength) {
+        unsafe {
+            sb::C_SkSVGSVG_setY(self.native_mut(), *y.native());
+        }
+    }
+
+    pub fn set_width(&mut self, width: DomSVGLength) {
+        unsafe {
+            sb::C_SkSVGSVG_setWidth(self.native_mut(), *width.native());
+        }
+    }
+
+    pub fn set_height(&mut self, height: DomSVGLength) {
+        unsafe {
+            sb::C_SkSVGSVG_setHeight(self.native_mut(), *height.native());
+        }
+    }
+}
+
+#[repr(C)]
+#[derive(Copy, Clone, PartialEq, Debug)]
+pub enum SvgUnit {
```

Comment:
nit: I would rather see this directly taken from the bindings and just `use SkSVGLength_Unit as Unit` an add a rule (see  skia_bindgen.rs ENUM_REWRITES) to strip the `k` prefix from the variant names. This way we don't have to adjust the Rust code if members change.